### ISSUE TITLE
Investigate reports of records getting lost when sim cards are changed 

### ISSUE THREAD

**Issue by TNgidi** [disarm-platform/user-requests-and-feedback#109]
Records getting lost when sim cards are changed on tablets. Reported by @sbabur on an email

**Comment by TNgidi** [comment]
We are trying to reproduce it on our side

**Comment by sbabur** [comment]
@TNgidi  Great thanks! I don't have another sim card to test this scenario out, or another kind soul willing to lend me theirs

**Comment by sbabur** [comment]
@TNgidi Were you able to test out this scenario?

**Comment by TNgidi** [comment]
@sbabur Yes I tested this but was not able to reproduce the error. I made records with the sim card inserted, then put in a different SIM card and the records were still in the device waiting to be synched.

**Comment by sbabur** [comment]
@TNgidi Noted, thanks - I will test later this week when I get to zim as well
